File: filedata/entity/c#/Videoplayconfig.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("videoplay_config")]
    public class Videoplayconfig
    {
        
        /// <summary>
        /// ID主键 这是视频播放配置表
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 短视频播放域名
        /// </summary>
        [Column("ShortPlayUrl")]
        public string ShortPlayUrl { get; set; }
    
    
        /// <summary>
        /// 长视频播放域名
        /// </summary>
        [Column("LongPlayUrl")]
        public string LongPlayUrl { get; set; }
    
    
        /// <summary>
        /// 专辑播放域名
        /// </summary>
        [Column("AlbumPlayUrl")]
        public string AlbumPlayUrl { get; set; }
    
    
        /// <summary>
        /// 状态 0=停用 1=启用
        /// </summary>
        [Column("Status")]
        public bool Status { get; set; }
    
    
        /// <summary>
        /// 是否站点专属域名 0=不是 1=是
        /// </summary>
        [Column("IsPrivate")]
        public bool IsPrivate { get; set; }
    
    
        /// <summary>
        /// 专属站点
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
    }
}

File: filedata/entity/java/videoplayConfig.java

package entity;
// 
public class Videoplayconfig  extends BaseDomain{
    public Videoplayconfig() {

    }
    
    //
    private String Id;
        
    //
    private String ShortPlayUrl;
        
    //
    private String LongPlayUrl;
        
    //
    private String AlbumPlayUrl;
        
    //
    private bit(10) Status;
        
    //
    private bit(10) IsPrivate;
        
    //
    private String CompanyStyle;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public String getShortplayurl() {
        return Shortplayurl;
    }
    public void setShortplayurl(String Shortplayurl) {
        this.Shortplayurl = Shortplayurl;
    }
            
    public String getLongplayurl() {
        return Longplayurl;
    }
    public void setLongplayurl(String Longplayurl) {
        this.Longplayurl = Longplayurl;
    }
            
    public String getAlbumplayurl() {
        return Albumplayurl;
    }
    public void setAlbumplayurl(String Albumplayurl) {
        this.Albumplayurl = Albumplayurl;
    }
            
    public bit(10) getStatus() {
        return Status;
    }
    public void setStatus(bit(10) Status) {
        this.Status = Status;
    }
            
    public bit(10) getIsprivate() {
        return Isprivate;
    }
    public void setIsprivate(bit(10) Isprivate) {
        this.Isprivate = Isprivate;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
             
    @Override
    public String toString() {
        return "videoplay_config[Id=" + Id + ",ShortPlayUrl=" + ShortPlayUrl + ",LongPlayUrl=" + LongPlayUrl + ",AlbumPlayUrl=" + AlbumPlayUrl + ",Status=" + Status + ",IsPrivate=" + IsPrivate + ",CompanyStyle=" + CompanyStyle + "]";
    }
        
    
}
        